base/database/etl_wormbase.py

```diff
@@ -98,17 +103,25 @@ def fetch_orthologs(orthologs_fname: str):
     """
     csv_out = list(csv.reader(open(orthologs_fname, 'r'), delimiter='\t'))
 
+    idx = 0
+    count = 0
     for line in csv_out:
-        size_of_line = len(line)
-        if size_of_line < 2:
-            continue
-        elif size_of_line == 2:
-            wb_id, locus_name = line
-        else:
-            yield {'gene_id': wb_id,
-                   'gene_name': locus_name,
-                   'homolog_species': line[0],
-                   'homolog_taxon_id': None,
-                   'homolog_gene': line[2],
-                   'homolog_source': line[3],
-                   'is_ortholog': line[0] == 'Caenorhabditis elegans'}
+      idx += 1
+      size_of_line = len(line)
+      if size_of_line < 2:
+        continue
+      elif size_of_line == 2:
+        wb_id, locus_name = line
+      else:
+        ref = WormbaseGeneSummary.query.filter(WormbaseGeneSummary.gene_id == wb_id).first()
+        if idx % 10000 == 0:
+          logger.info(f'Processed {idx} records yielding {count} inserts')
+        if ref:
+          count += 1
+          yield {'gene_id': wb_id,
+                 'gene_name': locus_name,
+                 'homolog_species': line[0],
+                 'homolog_taxon_id': None,
+                 'homolog_gene': line[2],
+                 'homolog_source': line[3],
+                 'is_ortholog': line[0] == 'Caenorhabditis elegans'}
```

Comment:
naive question, but should this be case insensitive?

Reply:
The data source it's referencing is standardized